stdlib/Random/src/misc.jl

```diff
@@ -220,6 +220,20 @@ function shuffle!(r::AbstractRNG, a::AbstractArray)
     return a
 end
 
+function shuffle!(r::AbstractRNG, a::AbstractArray{Bool})
```

Comment:
I guess this should be

```suggestion
function shuffle!(r::AbstractRNG, v::AbstractArray{Bool})
```

?